controllers/validation/validation.go

```diff
@@ -76,3 +85,27 @@ func validateArbiterSpec(mdb mdbv1.MongoDBCommunity) error {
 
 	return nil
 }
+
+// validateAuthModeSpec checks that the list of modes does not contain duplicates.
+func validateAuthModeSpec(mdb mdbv1.MongoDBCommunity) error {
+	allModes := mdb.Spec.Security.Authentication.Modes
+
+	// if no mode is specified in the specs, it's valid
+	if len(allModes) == 0 {
+		return nil
+	}
```

Comment:
I believe modes is a required CRD field, so we actually will require at least one entry!

Reply:
Maybe you can double check this by trying to create a resource manually without specifying an entry for `authentication.modes`